packages/language/src/preprocessor/pli-preprocessor-parser.ts

```diff
@@ -191,8 +191,11 @@ export class PliPreprocessorParser {
       case PreprocessorTokens.Skip.tokenTypeIdx:
         unit = this.skipStatement(state);
         break;
+      case PreprocessorTokens.XInclude.tokenTypeIdx:
+        unit = this.includeStatement(state, { xinclude: true });
+        break;
```

Comment:
Suggestion: The `includeStatement` should by itself decide whether it wants to parse a `XINCLUDE` or `INCLUDE`. Passing this info into the parsing method just makes everything more complicated. The same applies to the `GOTO`/`GO TO` statement below. 
```suggestion
      case PreprocessorTokens.XInclude.tokenTypeIdx:
      case PreprocessorTokens.Include.tokenTypeIdx:
        unit = this.includeStatement(state);
        break;
```